src/bloqade/squin/rewrite/qasm2/noise_to_squin.py

```diff
@@ -0,0 +1,80 @@
+from kirin import ir
+from kirin.dialects import func
+from kirin.rewrite.abc import RewriteRule, RewriteResult
+
+from bloqade import squin
+from bloqade.qasm2.dialects.noise import stmts as noise_stmts
+
+from .util import num_to_py_constant
```

Comment:
Do we have to create a `util` file for a single function used in a single file? 😅